tools/tests/tester/integ/TestRunner.test.ts

```diff
@@ -29,19 +29,20 @@ test('stories folder', async () => {
   }
 
   const expected_evaluations = [
-    'passed',
     'error/chapter_error',
     'error/output_error',
     'error/prologue_error',
     'failed/invalid_data',
     'failed/not_found',
+    'passed/passed',
+    'passed/value_type',
     'skipped/semver',
     'skipped/distributions/chapters',
     'skipped/distributions/excluded',
     'skipped/distributions/included'
   ].map((fixture) => { return load_expected_evaluation(fixture, true) })
 
-  expect(actual_evaluations).toStrictEqual(expected_evaluations)
+  expect(actual_evaluations).toEqual(expected_evaluations)
```

Comment:
This is because of this:

```
   @@ -959,11 +959,11 @@
              "title": "DELETE /movies",
            },
          ],
          "prologues": Array [
            Object {
    -         "output": Object {
    +         "output": ChapterOutput {
                "outputs": Object {
                  "boolean": false,
                },
              },
              "overall": Object {
```